src/admin/cli.py

```
@@ -0,0 +1,256 @@
+def get_bridge_manager() -> BridgeManager:
+    management_room = os.environ.get("KNARR_MANAGEMENT_ROOM")
+    if not management_room:
+        click.echo("Error: KNARR_MANAGEMENT_ROOM is required.", err=True)
+        sys.exit(1)
+    return BridgeManager(get_bridge_client(), management_room)
+
+
+@click.group()
+def cli():
+    """Knarr — operational commands for Matrix homeserver and bridges."""
+    pass
```

Ruff (0.15.10)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)